apiary.apib

```diff
@@ -3267,6 +3267,8 @@ Gets the users with the specified feature assigned.
 
 ### Create a feature [POST]
 
+**Please, if your feature is intended for all stacks, implement it through migration.**
```

Comment:
```suggestion
**Unless there is a specific reason why the feature MUST NOT be present on some stack, implement it through migration.**
```

Protože jinak to skoro nikdo nepoužije, protože si řekne, že to nechce zapínat na single tenantech IMHO